ROADMAP.md

```diff
@@ -1,7 +1,27 @@
 # Kubeflow Roadmap
 
-## Kubeflow 1.7 Release - planning now in process 
-* Please join the Kubeflow Community Meetings on Tuesdays for updates and for opportunities to contribute.  
+## Kubeflow 1.7 Release, Planned: March 2023 
+The Kubeflow Community plans to deliver its v1.7 release in March 2023, per this [timeline](https://github.com/kubeflow/community/pull/573).   The high level deliveries are being tracked in this [Project Board](https://github.com/orgs/kubeflow/projects/50/views/1).   The v1.7 release process will be managed by the v1.7 [Release Team](https://github.com/kubeflow/internal-acls/pull/576) using the best practices in the [Release Handbook](https://github.com/kubeflow/community/blob/master/releases/handbook.md)
+
+Notable feature candidates in the [Project Board](https://github.com/orgs/kubeflow/projects/50/views/1) are:
+* Support for Kubernetes 1.25
+* Improved user isolation especially for the Kubeflow pipelines user interface, database, and artifacts
+* Update Kubeflow Notebooks naming from Notebooks to Workbenches
+* Delivery of KFP V2 beta with its new front-end, backend and SDK 
+* Simplified creation of Katib and Training Operator experiments using SDKs
+* Simplified hyperparameter trial and log access.  Katib plans to add filtering and sorting of hyperparameter trial lists, and to speed user debugging by exposing logs from previous and running trials.
```

Comment:
These features to be added in UI

Reply:
@johnugeorge I have updated the description to reference that the functionality is available in the UI.